l10n_br_fiscal/models/icms_regulation.py

```diff
@@ -18,10 +18,10 @@
 <page name="uf_{0}" string="{1}">
     <notebook>
         <page name="uf_{0}_internal" string="Interno">
-            <group name="icms_internal_{0}" string="Internal">
+            <group name="icms_internal_{0}" string="Interno">
```

Comment:
@gabrielcardoso21 Do you have a specific reason to switch to the string for Portuguese?

Reply:
I couldn't translate it in the .po file. If there's a way to do this I can remove this commit.

Reply:
Mas é um problema de tradução comum nas tags group ou somente neste caso especifico ?

https://github.com/odoo/odoo/blob/e7bfadea847012ee956b7bb425688b5f9ccda9cf/addons/account/views/partner_view.xml#L229
https://github.com/odoo/odoo/blob/e7bfadea847012ee956b7bb425688b5f9ccda9cf/addons/account/i18n/account.pot#L4281

Reply:
I couldn't translate because it is a dynamicly created view, I'll remove the commit, but if any of you know how to proceed in such cases, please comment here the solution. Thanks.